go.mod

```diff
@@ -3,22 +3,20 @@ module github.com/harness/ff-golang-server-sdk
 go 1.16
 
 require (
-	github.com/deepmap/oapi-codegen v1.6.0
-	github.com/fatih/structtag v1.2.0 // indirect
-	github.com/getkin/kin-openapi v0.53.0
-	github.com/golang-jwt/jwt v3.2.1+incompatible
-	github.com/google/uuid v1.2.0
+	github.com/deepmap/oapi-codegen v1.11.0
```

Comment:
I think we should be careful when increasing dep versions, this module is used with ff-server so probably it will need some sanity check